components/Footer/Footer.tsx

```diff
@@ -18,13 +18,19 @@ export const Footer = () => {
 							href="https://github.com/AccessibleForAll/AccessibleWebDev/blob/main/CONTRIBUTING.md"
 							className={styles.footerLink}>
 							Become a Contributor
+							<span>
+								<BsBoxArrowUpRight aria-label="opens in new window" />
+							</span>
 						</a>
 					</li>
 					<li>
 						<a
 							href="https://github.com/sponsors/EmmaDawsonDev"
 							className={styles.footerLink}>
 							Support Us
+							<span>
+								<BsBoxArrowUpRight aria-label="opens in new window" />
+							</span>
```

Comment:
```suggestion
							Support Us{" "}
							<span>
								<BsBoxArrowUpRight aria-label="opens in new window" />
							</span>
```

Reply:
i modified the footer CSS file because of the space. but I'll add it the way you suggested.

Reply:
Ah, I didn't realise this was the change you made in the CSS file, it looked just like formatting changes. However, this method is more robust for people that might zoom in as the space will follow the same font-sizing as the surrounding text.